src/components/testimonial/Testimonial.css

```diff
@@ -0,0 +1,121 @@
+.testimonial-card {
+  display: flex;
+  flex-direction: column;
+  border-radius: .3rem;
+  flex-basis: 20rem;
+  padding: 1rem;
+  max-width: 20rem;
+  flex-grow: 1;
+  row-gap: .7rem;
+  border-bottom-left-radius: 0;
+  border-bottom-right-radius: 0;

```

Comment:
Deze twee properties zijn nu in strijd met `border-radius: .3rem;` die je een paar regels hoger hebt neergezet. Als je verschillende radiussen wil toewijzen, kun je ook border-radius `border-radius: .3rem .3rem 0 0;` gebruiken!